src/Lib/Items/Model.vala

```diff
@@ -381,6 +379,8 @@ public class Akira.Lib.Items.Model : Object {
 
         new_node.parent = parent_node;
 
+        add_to_maps (new_node, true);
+
```

Comment:
@mbfraga I had to move the adding of the node to the canvas after the parent was assigned in order to create layers in the proper position.
Is this okay? It doesn't seem to affect performance or anything else/